src/pyfaf/queries.py

```diff
@@ -1287,6 +1288,17 @@ def get_crashed_unknown_package_nevr_for_report(db, report_id):
             .filter(st.ReportUnknownPackage.type == "CRASHED")
             .all())
 
+def get_crashed_package_version_for_report(db, report_id):
+    """
+    Return version for unknown packages that CRASHED in a given
+    report.
+    """
+    return (db.session.query(st.ReportUnknownPackage.version)
```

Comment:
ReportUnknownPackage is not enough (I know it works on your system well, but in general will not work) You should select from ReportPackage and from ReportUnknownPackage